src/VisualStudio/Core/Def/Implementation/Remote/RemotableDataJsonRpcEx.cs

```diff
@@ -122,7 +122,7 @@ private Task WriteNoAssetAsync(ObjectWriter writer)
 
         private async Task WriteOneAssetAsync(ObjectWriter writer, int scopeId, Checksum checksum, CancellationToken cancellationToken)
         {
-            var remotableData = _remotableDataService.GetRemotableData(scopeId, checksum, cancellationToken) ?? RemotableData.Null;
+            var remotableData = await _remotableDataService.GetRemotableDataAsync(scopeId, checksum, cancellationToken).ConfigureAwait(false) ?? RemotableData.Null;
```

Comment:
💡 The relative order of operations between `await` and `??` is not obvious here. Parentheses for explicit precedence would help readability.

Reply:
I need to go through code but GetRemotableData probably never needs to be Async except for the Text. so GetRemotableData should return ValueTask